### ISSUE TITLE
Installing Seurat on Action Of The Toes

### ISSUE THREAD

**Issue by bennigoetz** [satijalab/seurat#2051]
Installation doesn't seem to work for the latest R version.

```R
> install.packages('Seurat')
--- Please select a CRAN mirror for use in this session ---
...
Warning: unable to access index for repository https://cran.revolutionanalytics.com/src/contrib:
  cannot open URL 'https://cran.revolutionanalytics.com/src/contrib/PACKAGES'
Warning message:
package ‘Seurat’ is not available (for R version 3.6.1)
```

**Comment by miesav** [comment]
This seems like your server is having issues accessing the Repo or the Repo might be down. Try to access this in your browser: 

https://cran.revolutionanalytics.com/src/contrib/PACKAGES

If it does not work try a different mirror:

install.packages('Seurat', repos='https://cloud.r-project.org/src/contrib/PACKAGES')

**Comment by bennigoetz** [comment]
Thanks for the quick reply. The URL suggested I try doesn't load the page, it just sits there. So I tried the different mirror for install.packages, and got this error:

```R
> install.packages('Seurat', repos='https://cloud.r-project.org/src/contrib/PACKAGES')
Warning: unable to access index for repository https://cloud.r-project.org/src/contrib/PACKAGES/src/contrib:
  cannot open URL 'https://cloud.r-project.org/src/contrib/PACKAGES/src/contrib/PACKAGES'
Warning message:
package ‘Seurat’ is not available (for R version 3.6.1)
```

*However*, this worked (same idea):

```R
install.packages('Seurat', repos='http://cran.us.r-project.org')
```

Thanks for the fast response, even though this was a general R problem, not something Seurat-specific.
